src/cf.data.pre

```diff
@@ -1043,6 +1043,16 @@ DOC_START
 	  # NOTE 2: IPv6 protocol does not contain ARP. MAC/EUI is either
 	  # encoded directly in the IPv6 address or not available.
 
+	acl aclname connmark mark[/mask] ...
```

Comment:
AFAICT, the new code can only match markings on the client-Squid connections. If markings are possible on Squid-origin/peer connections as well, then we need a different name for this new ACL, right? Please note that it should not matter whether those Squid-server connection markings are set by Squid itself or something else.

Reply:
Maybe clientside_connmark or client_connmark will be a better option?